### PR TITLE
[chore/#46]:metaTag 설정 및 추가

### PR DESCRIPTION
**Pull request by juhee067**
## 📌 작업 내용

metaTag 설정 및 추가
```tsx
import MetaTag from '@/components/MetaTag';
import { ProductGrid } from '@/features/Catalog';
import Filter from '@/features/Catalog/components/Filter';
import styled from 'styled-components';

function CatalogPage() {
  return (
    <>
      <MetaTag
        title='리쿠르트 - 전통주 카탈로그'
        description='다양한 전통주를 찾아보고 필터링하여 원하는 상품을 쉽게 찾아보세요.'
        keywords='리쿠르트, 전통주, 카탈로그, 술, 상품'
        url='https://www.licruit.site/catalog'
      />
      <Content>
        <div className='aside'>
          <Filter />
        </div>
        <div className='article'>
          <ProductGrid />
        </div>
      </Content>
    </>
  );
}

export default CatalogPage;

`;

```
<br/>

## 📌 구현 결과 (선택)

image

<br/>

### CODE REVIEW

**Comment by JIMIN1020** [comment]
<h2>Coverage Report</h2>
<table> <thead> <tr> <th align="center">Status</th> <th align="left">Category</th> <th align="right">Percentage</th> <th align="right">Covered / Total</th> </tr> </thead> <tbody> <tr> <td align="center">🔵</td> <td align="left">Lines</td> <td align="right">0%</td> <td align="right">0 / 4618</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Statements</td> <td align="right">0%</td> <td align="right">0 / 4618</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Functions</td> <td align="right">3.61%</td> <td align="right">6 / 166</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Branches</td> <td align="right">3.61%</td> <td align="right">6 / 166</td> </tr> </tbody> </table><details><summary>File Coverage</summary><table> <thead> <tr> <th align="left">File</th> <th align="right">Stmts</th> <th align="right">% Branch</th> <th align="right">% Funcs</th> <th align="right">% Lines</th> <th align="left">Uncovered Lines</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6"><b>Changed Files</b></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/components/MetaTag.tsx">src/components/MetaTag.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/components/MetaTag.tsx#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/components/MetaTag.tsx#L10-L11" class="text-red">10-11</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/components/MetaTag.tsx#L13-L15" class="text-red">13-15</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/components/MetaTag.tsx#L17-L18" class="text-red">17-18</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/components/MetaTag.tsx#L20-L25" class="text-red">20-25</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/components/MetaTag.tsx#L27-L28" class="text-red">27-28</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/components/MetaTag.tsx#L30" class="text-red">30</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/components/MetaTag.tsx#L32" class="text-red">32</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatalogPage.tsx">src/pages/CatalogPage.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatalogPage.tsx#L1-L4" class="text-red">1-4</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatalogPage.tsx#L6-L23" class="text-red">6-23</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatalogPage.tsx#L25" class="text-red">25</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatalogPage.tsx#L27" class="text-red">27</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatalogPage.tsx#L29-L30" class="text-red">29-30</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatalogPage.tsx#L41" class="text-red">41</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatelogDetailPage.tsx">src/pages/CatelogDetailPage.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatelogDetailPage.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatelogDetailPage.tsx#L5-L16" class="text-red">5-16</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatelogDetailPage.tsx#L18" class="text-red">18</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatelogDetailPage.tsx#L20" class="text-red">20</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/CatelogDetailPage.tsx#L22" class="text-red">22</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/FindPasswordPage.tsx">src/pages/FindPasswordPage.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/FindPasswordPage.tsx#L1-L4" class="text-red">1-4</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/FindPasswordPage.tsx#L9-L10" class="text-red">9-10</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/FindPasswordPage.tsx#L12-L14" class="text-red">12-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/FindPasswordPage.tsx#L16-L31" class="text-red">16-31</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/FindPasswordPage.tsx#L33-L40" class="text-red">33-40</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/FindPasswordPage.tsx#L42" class="text-red">42</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/FindPasswordPage.tsx#L44" class="text-red">44</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/LoginPage.tsx">src/pages/LoginPage.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/LoginPage.tsx#L1-L7" class="text-red">1-7</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/LoginPage.tsx#L9-L10" class="text-red">9-10</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/LoginPage.tsx#L12-L27" class="text-red">12-27</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/LoginPage.tsx#L30-L33" class="text-red">30-33</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/LoginPage.tsx#L35" class="text-red">35</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/LoginPage.tsx#L37" class="text-red">37</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/LoginPage.tsx#L39" class="text-red">39</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/LoginPage.tsx#L43-L44" class="text-red">43-44</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/LoginPage.tsx#L47" class="text-red">47</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/MainPage.tsx">src/pages/MainPage.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/MainPage.tsx#L1-L6" class="text-red">1-6</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/MainPage.tsx#L8-L9" class="text-red">8-9</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/MainPage.tsx#L11-L14" class="text-red">11-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/MainPage.tsx#L16-L31" class="text-red">16-31</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/MainPage.tsx#L33" class="text-red">33</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/MainPage.tsx#L35" class="text-red">35</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/SignUpPage.tsx">src/pages/SignUpPage.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/SignUpPage.tsx#L1-L4" class="text-red">1-4</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/SignUpPage.tsx#L6-L10" class="text-red">6-10</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/SignUpPage.tsx#L12-L14" class="text-red">12-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/SignUpPage.tsx#L16" class="text-red">16</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/SignUpPage.tsx#L20-L27" class="text-red">20-27</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/SignUpPage.tsx#L29-L38" class="text-red">29-38</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/SignUpPage.tsx#L40-L62" class="text-red">40-62</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/SignUpPage.tsx#L64-L71" class="text-red">64-71</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/SignUpPage.tsx#L73" class="text-red">73</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/5926c410f03025393c2c912693a17dc7a475c459/src/pages/SignUpPage.tsx#L75" class="text-red">75</a></td> </tr> </tbody> </table></details>
<em>Generated in workflow <a href=https://github.com/Licruit/Licruit-frontend/actions/runs/10380994376>#102</a></em>

**Review by JIMIN1020 — APPROVED** [review]
수고 많으셨습니다!!

**Review by SebellKo — APPROVED** [review]
우리도 헬멧 썼네요 ㅎㅎ
고생하셨어요 주히님

File: src/components/MetaTag.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { Helmet } from 'react-helmet-async';
+
+interface Props {
+  title: string;
+  description: string;
+  keywords: string;
+  imgsrc?: string;
+  url: string;
+}
+function MetaTag(Meta: Props) {
```
Comment (ksy90101) [inline src/components/MetaTag.tsx:10]: default값이 있으면 어떨까요?

File: src/pages/CatalogPage.tsx [hunk 2]
```diff
@@ -1,10 +1,19 @@
+import MetaTag from '@/components/MetaTag';
 import { ProductGrid } from '@/features/Catalog';
 import Filter from '@/features/Catalog/components/Filter';
 import styled from 'styled-components';
 
 function CatalogPage() {
   return (
     <>
+
```
Comment (ksy90101) [inline src/pages/CatalogPage.tsx:9]: CI가 깨지는데요??

File: src/pages/MainPage.tsx [hunk 3]
```diff
@@ -14,6 +15,13 @@ function MainPage() {
 
   return (
     <>
+      <MetaTag
+        title='리쿠르트 - 전통주와 함께하는 새로운 경험'
+        description='리쿠르트 메인 페이지에서 전통주와 관련된 다양한 정보를 만나보세요.'
```
Comment (ksy90101) [inline src/pages/MainPage.tsx:20]: 중복되니 Default값을 지정해두어도 좋을거 같네요.

**Review by ksy90101 — COMMENTED** [review]
Good!

**Pull request merged**